text2digits/text2digits.py

```diff
@@ -1,275 +1,134 @@
-import re
-import math
+from typing import List
 
-UNITS = ['zero', 'one', 'two', 'three', 'four', 'five', 'six', 'seven', 'eight', 'nine']
-TEENS = ['ten', 'eleven', 'twelve', 'thirteen', 'fourteen', 'fifteen', 'sixteen', 'seventeen', 'eighteen', 'nineteen']
-TENS = ['ten', 'twenty', 'thirty', 'forty', 'fifty', 'sixty', 'seventy', 'eighty', 'ninety', 'hundred', 'thousand', 'million', 'billion']
-SCALES = ['hundred', 'thousand', 'million', 'billion', 'trillion']
-ORDINAL_WORDS = {'oh': 0, 'first': 1, 'second': 2, 'third': 3, 'fourth': 4, 'fifth': 5, 'sixth': 6, 'seventh': 7, 'eighth': 8, 'ninth': 9, 'twelfth': 12}
-ORDINAL_ENDINGS = [('ieth', 'y'), ('th', '')]
+from text2digits.tokens_basic import Token, WordType
+from text2digits.rules import CombinationRule, ConcatenationRule
+from text2digits.text_processing_helpers import split_glues, find_similar_word
 
 
-class TextPreprocessor:
-    @staticmethod
-    def bigram(word1, word2):
-        '''
-        Returns the bigram similarity of 2 words
-        '''
-        word1 = word1.lower()
-        word2 = word2.lower()
-        word1_length = len(word1)
-        word2_length = len(word2)
-        pairs1 = []
-        pairs2 = []
+class Text2Digits(object):
+    def __init__(self, similarity_threshold=1.0, convert_ordinals=True, add_ordinal_ending=False):
+        """
+        This class can be used to convert text representations of numbers to digits. That is, it replaces all occurrences of numbers (e.g. forty-two) to the digit representation (e.g. 42).
 
-        for i in range(word1_length):
-            if i == word1_length - 1:
-                continue
-            pairs1.append(word1[i] + word1[i + 1])
+        Basic usage:
 
-        for i in range(word2_length):
-            if i == word2_length - 1:
-                continue
-            pairs2.append(word2[i] + word2[i + 1])
+        >>> from text2digits import text2digits
```

Comment:
Remove these 4 lines

Reply:
You mean the basic usage example from the class documentation? Why? Or is something wrong with it?

Reply:
Sorry I completely misread this yesterday. This is perfectly fine!